spaceship.zsh

```diff
@@ -16,19 +16,13 @@ SPACESHIP_PROMPT_TRUNC="${SPACESHIP_PROMPT_TRUNC:-3}"
 
 # PREFIXES
 SPACESHIP_PREFIX_SHOW="${SPACEHIP_PREFIX_SHOW:-true}"
-SPACESHIP_PREFIX_GIT="${SPACESHIP_PREFIX_GIT:-on}"
-SPACESHIP_PREFIX_HOST="${SPACESHIP_PREFIX_HOST:-at}"
-SPACESHIP_PREFIX_USER="${SPACESHIP_PREFIX_USER:-in}"
+SPACESHIP_PREFIX_MACHINE="${SPACESHIP_PREFIX_MACHINE:-at}"
```

Comment:
I think `SPACESHIP_PREFIX_HOST` and `SPACESHIP_PREFIX_DIR` would be much more consistence and obvious names for these variables. Please rename here and update the docs.